src/OpenIddict.Core/IOpenIddictStore.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Identity;
+
+namespace OpenIddict {
+    public interface IOpenIddictStore<TUser, TApplication> : IUserStore<TUser> where TUser : class where TApplication : class {
```

Comment:
`IOpenIddictStore` inherits from `IUserStore` for conveniency :smile: 

In the current bits, we're not using `TUser` yet, but we'll ultimately need it to retrieve the authorizations corresponding to a given user.